translation-config.yml

```diff
@@ -1,6 +1,46 @@
+locales:
+  default: en_US
+  supported:
+    - ar
+    - de
+    - en_GB
+    - es
+    - fr_CA
+    - fr_FR
+    - it
+    - ja
+    - nl
+    - pl
+    - pt
+    - ru
+    - sv
+    - zh_CN
+    - zh_HK
+
 bundles:
   - path: .
     extension: properties
     default_locale: en_US
   - path: src/main/resources
     extension: json
+
+snapshot:
+  copy:
+    - en_GB
+
+io:
+  out:
+    generator: translations.extensions.io_xls.XlsExporter
+    name: translations
+    mapping:
```

Comment:
I don't think this mapping is necessarily needed. The XlsExporter, the way it works, is already an Appian opinionated way of doing things and I don't really see it being usable outside of how we do things. This logic should just go into the exporter itself and maybe comment it there? Would make the configuration much cleaner and keep the yml readable. We'd just need to add in the README what `exporter` and `importer` is and nothing more

Reply:
I have to disagree on that one. True, there are some elements that are Appian specific, but these should be eliminated or driven by configuration (implemented now for the immediate need of ICC). Also, the configuration for importer/exporter is not intended to be forced to other importer/exporters, that is the reason why the importer/exporter ABCs accept config and options as parameters; they can use these as mechanisms to configure themselves and adjust to runtime execution.

Reply:
My concern is mostly with the documentation that would need to go into explaining what this mapping actually is. It only makes sense if you understand the full workflow of what it means to export and import. It's not immediately clear what `mapping` means and what the consequences of it are. I want this tool to be easy on the documentation :)